### DISCUSSION TITLE
wildcard dns + prom-label-proxy

### DISCUSSION THREAD

**Discussion by kfox1111** [grafana/grafana#40423] [General]
Say I had a wildcard dns entry for
*.grafana.k8s.mysite.com pointing at a grafana instance. Say, also I had grafana configured to point at a prom-label-proxy filtering prom queries by namespace label.

Would it/could it be possible to pass the short hostname from dns used as the namespace parameter's value to prom-label-proxy?

This would allow one instance of grafana to selectively be used for multiple projects.

foo.grafana.k8s.mysite.com could only query prom for namespace=foo data.
bar.grafana.k8s.mysite.com could only query prom for namespace=bar data. etc etc.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
